frontend/src/metabase/api/dashboard.ts

```diff
@@ -0,0 +1,93 @@
+import type {
+  CopyDashboardRequest,
+  CreateDashboardRequest,
+  Dashboard,
+  DashboardId,
+  ListDashboardsRequest,
+  SaveDashboardRequest,
+  UpdateDashboardRequest,
+} from "metabase-types/api";
+
+import { Api } from "./api";
+import {
+  idTag,
+  invalidateTags,
+  listTag,
+  provideDashboardListTags,
+  provideDashboardTags,
+} from "./tags";
+
+export const dashboardApi = Api.injectEndpoints({
+  endpoints: builder => ({
+    listDashboards: builder.query<Dashboard[], ListDashboardsRequest | void>({
+      query: () => ({
+        method: "GET",
+        url: "/api/dashboard",
+      }),
```

Comment:
Where are we using the query function? Should it accept `body`?

```suggestion
    listDashboards: builder.query<Dashboard[], ListDashboardsRequest | void>({
      query: body => ({
        method: "GET",
        url: "/api/dashboard",
        body.
      }),
```

Reply:
if it's GET, I think it shouldn't

Reply:
> Should it accept `body`?

Yes, good catch :+1: 
5b5affa6651b7b9b9ca70beca7fb1bc53a5f193d

> if it's GET, I think it shouldn't

It actually should.